### DISCUSSION TITLE
Updating parameter files for Martini 3.0

### DISCUSSION THREAD

**Discussion by ghost** [marrink-lab/polyply_1.0#387] [Martini3]
Hi all,
I have parameter files for SMA and DIBMA from the following publications, from doi:10.1021/acs.langmuir.8b03978 and https://doi.org/10.1016/j.bbamem.2020.183207.
However these are both for Martini 2. I would love to update them to 3 and upload them to the Polyply database. However this is somewhat outside my wheelhouse. Does anyone have any good tutorials for updating parameters from Martini 2 to 3?
All the best,
Jonah

**Comment by fgrunewald** [comment]
Hi Johna, 

To update the polymers you'd have to update the bead assignment and bonded parameters to align them with Martini3. A tutorial that uses polyply and goes through the task can be found [here](https://ricalessandri.github.io/M3-Tutorials/polymers.html). Be aware that'd you'll need some all-atom simulations and most likely a new mapping. A library of current Martini3 polymers (e.g. Polystyrene) and some discussion can be in [here](https://research.rug.nl/en/publications/material-design-using-martini-accelerating-discovery-through-coar) in chapter 5. 

Be warned that this is not a super trivial task. But we are happy to help.

Cheers,
Fabian
